arch/arm/src/efm32/efm32_start.c

```diff
@@ -84,7 +84,7 @@ const uintptr_t g_idle_topstack = HEAP_BASE;
 
 #ifdef CONFIG_DEBUG_FEATURES
 #  if defined(CONFIG_ARMV7M_ITMSYSLOG)
-#    define showprogress(c) syslog_putc(c)
+#    define showprogress(c) up_putc(c)
```

Comment:
why?

Reply:
showprocess() is only called during the startup phase, up_putc is more suitable

Reply:
what's the point of this ifdef block then?

Reply:
FIX https://github.com/apache/nuttx/pull/14846